dev/how-to/migrate/from-mariadb-galera-cluster.md

```diff
@@ -0,0 +1,571 @@
+    | wsrep_local_recv_queue_max   | 1                                                 |
+    | wsrep_local_recv_queue_min   | 0                                                 |
+    | wsrep_local_replays          | 0                                                 |
+    | wsrep_local_send_queue       | 0                                                 |
+    | wsrep_local_send_queue_avg   | 0.000000                                          |
+    | wsrep_local_send_queue_max   | 1                                                 |
+    | wsrep_local_send_queue_min   | 0                                                 |
+    | wsrep_local_state            | 4                                                 |
+    | wsrep_local_state_comment    | Synced                                            |
+    | wsrep_local_state_uuid       | eb0f4303-2ea7-11e9-87ba-679c851caca2              |
+    | wsrep_open_connections       | 0                                                 |
+    | wsrep_open_transactions      | 0                                                 |
+    | wsrep_protocol_version       | 9                                                 |
+    | wsrep_provider_name          | Galera                                            |
+    | wsrep_provider_vendor        | Codership Oy <info@codership.com>                 |
+    | wsrep_provider_version       | 25.3.25(r3836)                                    |
+    | wsrep_ready                  | ON                                                |
+    | wsrep_received               | 30                                                |
+    | wsrep_received_bytes         | 9726                                              |
+    | wsrep_repl_data_bytes        | 6205                                              |
+    | wsrep_repl_keys              | 60                                                |
+    | wsrep_repl_keys_bytes        | 960                                               |
+    | wsrep_repl_other_bytes       | 0                                                 |
+    | wsrep_replicated             | 20                                                |
+    | wsrep_replicated_bytes       | 8496                                              |
+    | wsrep_thread_count           | 5                                                 |
+    +------------------------------+---------------------------------------------------+
+    61 rows in set (0.00 sec)
+    ```
+
+3. Run the `show global status like '%wsrep%';` command to make sure the following conditions are satisfied:
+
+    - `wsrep_local_state_commment` is `synced`.
+    - `wsrep_local_state` is 4.
+    - `swrep_cluster_status` is `primary`.
+    - Values of `wsrep_local_index`s are different.
+
+4. Log in to different nodes respectively, create testing tables, insert data, and observe the data replication status.
+
+## Migrate data to TiDB using Data Migration
+
+1. Deploy Data Migration.
```

Comment:
I don't think it makes sense to have such detailed instructions for deploying MariaDB/Galera and basically zero instructions for deploying TiDB. Presumably a user coming *from* MariaDB will already be using MariaDB but will *not* be using TiDB?